--- tmp/sendip-2.5/Makefile	2013-08-30 21:56:36.591283235 +0200
+++ Makefile	2016-06-08 19:26:40.622555391 +0200
@@ -9,7 +9,7 @@
 #INSTALL=/usr/ucb/install
 
 CFLAGS=	-fPIC -fsigned-char -pipe -Wall -Wpointer-arith -Wwrite-strings \
-			-Wstrict-prototypes -Wnested-externs -Winline -Werror -g -Wcast-align \
+			-Wstrict-prototypes -Wnested-externs -Winline -g -Wcast-align \
 			-DSENDIP_LIBS=\"$(LIBDIR)\"
 #-Wcast-align causes problems on solaris, but not serious ones
 LDFLAGS=	-g -rdynamic -lm
